prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -565,6 +567,22 @@ class Hl7Serializer(val metadata: Metadata) : Logging {
             }
         }
 
+        // check for alt CLIA for out of state testing
+        if (!hl7Config?.cliaForOutOfStateTesting.isNullOrEmpty()) {
+            val testingStateField = "OBX-24-4"
+            val pathSpecTestingState = formPathSpec(testingStateField)
+            val testingState = terser.get(pathSpecTestingState)
+
+            val settings = FileSettings(FileSettings.defaultSettingsDirectory)
```

Comment:
This could potentially lead to performance issues and bugs. The biggest issue here is that in production we load the organization information from the database, and not from files. If you look at `WorkflowEngine` you'll see it does a check to see if the `FEATURE_FLAG_SETTINGS_ENABLED` is true, and if it is it loads from the DB instead. I'd like you to change the logic here and add `FileSettings` as a param we pass into the `HL7Serializer`, and then in `WorkflowEngine` pass it in like `metadata` is. Make it a nullable parameter, and you can mock it in in your unit tests.